configs/anvil/config.20170926.FCT2.A_WCYCL1850S.ne30_oECv3.anvil

```diff
@@ -0,0 +1,148 @@
+[runs]
+## options related to the run to be analyzed and reference runs to be
+## compared against
+
+# mainRunName is a name that identifies the simulation being analyzed.
+mainRunName = 20170926.FCT2.A_WCYCL1850S.ne30_oECv3.anvil
+# preprocessedReferenceRunName is the name of a reference run that has been
+# preprocessed to compare against (or None to turn off comparison).  Reference
+# runs of this type would have preprocessed results because they were not
+# performed with MPAS components (so they cannot be easily ingested by
+# MPAS-Analysis)
+preprocessedReferenceRunName = B1850C5_ne30_v0.4
+
+[input]
+## options related to reading in the results to be analyzed
+
+# directory containing model results
+baseDirectory = /lcrc/group/acme/jwolfe/acme_scratch/20170926.FCT2.A_WCYCL1850S.ne30_oECv3.anvil/run
+
+# names of ocean and sea ice meshes (e.g. EC60to30, QU240, RRS30to10, etc.)
+mpasMeshName = oEC60to30v3
+
+[output]
+## options related to writing out plots, intermediate cached data sets, logs,
+## etc.
+
+# directory where analysis should be written
+baseDirectory = /dir/to/analysis/output
+
+# provide an absolute path to put HTML in an alternative location (e.g. a web
+# portal)
```

Comment:
do we have a more specific location for an html directory on anvil? pinging @vanroekel as well.

Reply:
not really.  We could maybe create one, but @jonbob and I have been using our lcrc scratch space for the output and then scp to nersc to the projectdir www directory

Reply:
I see. I'll check if there is any info online.

Reply:
I also just copied stuff over to NERSC just now (via my laptop).  I would honestly be surprised if LRCF has a data portal -- it seems pretty bare bones.  Maybe the solution is just to remove the commented out line here?